packages/velo-external-db/test/e2e/app.e2e.spec.js

```diff
@@ -1,30 +1,29 @@
 const { authOwner } = require('../drivers/auth_test_support')
-const each = require('jest-each').default
-const { initApp, teardownApp, dbTeardown, testSuits } = require('../resources/e2e_resources')
+const { initApp, teardownApp, dbTeardown } = require('../resources/e2e_resources')
+const { name, setup } = require('../resources/e2e_resources').testedSuit()
```

Comment:
why not just call the code inside of the describe, instead of here.
on the place that you will need the setup, just call it again.